configs/common/packages.yaml

```diff
@@ -184,11 +184,11 @@ packages:
     require: '@1.1.2'
   netcdf-c:
     # If using 4.9.1, turn off byterange variant to fix compile error: ~byterange
-    require: '@4.9.2 +dap +mpi ~parallel-netcdf ~szip build_system=autotools'
+    require: '@4.9.2 +dap +mpi ~parallel-netcdf ~szip +parallel_tests build_system=autotools'
```

Comment:
Can we add this variant for the weekly builds only, but have it off by default? How much does it add to the build time (if you know)? And will the tests run if the installation is from a binary cache, or only when the binary cache is updated?

Reply:
The variant only takes effect when `--test` is used, so this won't impact regular installations, so I'd prefer to do it in one place if you're okay with that. The build-time unit tests take a minute or two to run for each of netcdf-c and netcdf-fortran. The build-time tests are _not_ run when a package is installed from a binary cache.

Reply:
Ok, thanks for the clarification.